hw_5/task_1.py

```diff
@@ -0,0 +1,12 @@
+# Задача 26:  Напишите программу, которая на вход принимает два числа A и B,
+# и возводит число А в целую степень B с помощью рекурсии.
+# *Пример:*
+# A = 3; B = 5 -> 243 (3⁵)
+#     A = 2; B = 3 -> 8 
+
+def degree_ab(a, b):
+    if b <= 0 : #базовый случай
```

Comment:
Наверно, все-таки, просто
```suggestion
    if b == 0 : #базовый случай
```